src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -115,9 +115,53 @@ static bool _parseNumber(const char** content, float* number)
     if ((*content) == end) return false;
     //Skip comma if any
     *content = _skipComma(end);
+
+    return true;
+}
+
+
+static constexpr struct
+{
+    AspectRatioAlign align;
+    const char* tag;
+} alignTags[] = {
+    { AspectRatioAlign::XMinYMin, "xMinYMin" },
+    { AspectRatioAlign::XMidYMin, "xMidYMin" },
+    { AspectRatioAlign::XMaxYMin, "xMaxYMin" },
+    { AspectRatioAlign::XMinYMid, "xMinYMid" },
+    { AspectRatioAlign::XMidYMid, "xMidYMid" },
+    { AspectRatioAlign::XMaxYMid, "xMaxYMid" },
+    { AspectRatioAlign::XMinYMax, "xMinYMax" },
+    { AspectRatioAlign::XMidYMax, "xMidYMax" },
+    { AspectRatioAlign::XMaxYMax, "xMaxYMax" },
+};
+
+
+static bool _parseAspectRatio(const char** content, AspectRatioAlign* align, AspectRatioMeetOrSlice* meetOrSlice)
```

Comment:
Suggestion:  AspectRatioAlign _parseAspectRatio(const char** content, AspectRatioMeetOrSlice* meetOrSlice)
Here return value is useless?

Reply:
1. I'm not a fun of returning one value and the other one changing by passing it by ref/pointer. Now both are returned in the same way.
2. I think that this function should take care of all the text passed as the preserveAspectRatio value. so if in some svg file we get ```preserveAspectRatio="none some_additional_text"``` the return value should be false. So for now it's not usefull, but in the future, we could use the returned bool. That's why I didn;t use void.

Reply:
In the future when? I believe we can replace this in the future *when it's really necessary*. that's our policy.